apps/tcp_tunnel_device/src/tcp_tunnel_init.c

```diff
@@ -133,22 +182,47 @@ bool create_state_interactive_custom(const char* file) {
     printf("Enable Password Open Pairing: ");
     enablePasswordOpenPairing = yes_no();
 
+    printf(NEWLINE);
+    if (!enableLocalInitialPairing && !enablePasswordInvitePairing) {
+        printf("Both Local Initial Pairing and Password Invite Pairing modes are disabled. This means it will not be possible to create an Administrator of this device." NEWLINE);
+        printf("If IAM management is not needed, this is perfectly fine." NEWLINE);
```

Comment:
elaborate a bit more - "if (remote) management of users and access control is not needed"

Reply:
I do not understand your sentence. `(remote)` ?  and the construction with `management of` does that apply to both `users` and `access control`. Or are you under the impression that access control is non-existent in this case?

I have deliberately been very brief in these explanations to not have too much text and under the assumption that users has a basic understanding from the docs site. But I am happy to change the wording if you can explain what you think is missing.